.github/CONTRIBUTING.md

```diff
@@ -19,11 +19,47 @@ In our documentation, we label the "Explanation" section "Products", but otherwi
   background knowledge.
 - **Reference**: API documentation for the products.
 
-### Directory Structure
+### Directory Structure And Standards
 
-- The directory structure should match the final website structure.
+- The directory structure should match the final website structure, i.e. the path `/products/documentation` should be
+  represented by a file `products/documentation.mdx`.
 - Images should be placed in a sibling directory to the markdown file that references them, named `images`.
 
+```text
+products/sce/gateway
+├── images
+│   ├── container-gateway-error-400.png
+│   ├── container-gateway-error-401.png
+│   ├── container-gateway-error-403.png
+│   ├── container-gateway-error-404.png
+│   ├── container-gateway-error-408.png
+│   ├── container-gateway-error-410.png
+│   ├── container-gateway-error-413.png
+│   ├── container-gateway-error-500.png
+│   ├── container-gateway-error-501.png
+│   ├── container-gateway-error-502.png
+│   ├── container-gateway-error-503.png
+│   ├── container-gateway-error-504.png
+│   ├── load-balancer-options-1.png
+│   ├── portal-access-domain-name.png
+│   ├── portal-enable-authentication.png
+│   └── portal-enable-container-gateway.png
+├── enabling-ipv6.mdx
+├── error-pages.mdx
+├── load-balancer-options.mdx
+├── sending-requests.mdx
+└── websockets.mdx
+```
+
+- Files should be named with kebab-case, using full words. For example, `load-balancer-options.mdx` is preferred over
```

Comment:
I've never heard kebab-case before, but it was populated by copilot and I thought it was hilarious and descriptive